docs/concepts/deposits-withdrawals.md

```diff
@@ -0,0 +1,116 @@
+:::
+
+After a successful deposit transaction, wheter done directly or through Vega Console, the `Asset_Deposited` event will be emitted for use by the Vega event queue.
+
+The transaction is recognised by the Vega event queue and packaged as an event, which is then submitted to the validator nodes to verify the event contents against an Ethereum node that Vega validators also run.
+
+Once the transaction is verified, the Vega public key submitted in the transaction will be credited with the deposited asset.
+
+:::note Read more
+[ERC20 bridge logic API documentation](../api/bridge/contracts/ERC20_Bridge_Logic.md#deposit_asset)
+:::
+
+### Diagram: Deposits
+![Deposit diagram](/img/concept-diagrams/diagram-deposit.png)
+
+## Withdrawing assets
+**Assets used for trading** and related activities can only be withdrawn if they're in a general account. That means that, among other things, they can't be in bond for liquidity provision or in a margin account for active orders. 
+
+**VEGA tokens** can only be withdrawn if they are not staked and/or locked.
+
+To remove assets from the Vega network, submit a withdrawal request via a Vega app, such as the Vega Console trading interface, or the API.
+
+This request, if valid, will be put through consensus - the validators sign a multi-signature withdrawal order bundle for the ERC-20 bridge, and assign an expiry. The bridge validates the bundle and then releases the funds to the chosen Ethereum wallet.
+
+If the withdrawal isn't completed before the expiry, the tokens are returned to the participant's collateral account. 
+
+If it's a successful withdrawal transaction, the ERC20 bridge will emit an `Asset_Withdrawn` event, and confirms to the Vega network that the withdrawal has been completed.
+
+:::note Read more
+[ERC20 bridge logic API documentation](../api/bridge/contracts/ERC20_Bridge_Logic.md#withdraw_asset)
+:::
+
+### Withdrawal limits
+Withdrawals can have limits associated with them, where trying to withdraw above a certain amount will cause that withdrawal to be delayed by a set time. If a validator is compromised or otherwise issues a bad withdrawal, the delay gives Vega a chance to stop the withdrawal before it's too late.
+
+The two parts of a withdrawal limit are:
+* **Withdrawal threshold**: Set per asset through governance, requesting to withdraw that amount (and above) will trigger a withdrawal delay
+* **Withdrawal delay**: Set for all assets on the ERC-20 bridge, this is the time that a withdrawal is delayed before it's completed
+
+If you choose an amount to withdraw that is higher than the withdrawal threshold, the multi-signature bundle will only be usable after the withdrawal delay has passed, after which the assets can be moved into an Ethereum wallet.
+
+Once the delay time has passed, and the bundle is valid, **the withdrawal must be completed by the party**, such as by using Vega Console.
```

Comment:
```suggestion
Once the delay time has passed, and the bundle is valid, **the withdrawal must be completed by submitting the bundle to Ethereum and paying the gas fee required, usually this will be done by the party wishing to receive the funds**. This can be done using Vega Console or another frontend, or manually via the APIs and an Ethereum RPC node.
```